Documentation/api.md

```diff
@@ -850,6 +850,7 @@ ThanosSpec defines parameters for a Prometheus server within a Thanos deployment
 | baseImage | Thanos base image if other than default. Deprecated: use 'image' instead | *string | false |
 | resources | Resources defines the resource requirements for the Thanos sidecar. If not provided, no requests/limits will be set | [v1.ResourceRequirements](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#resourcerequirements-v1-core) | false |
 | objectStorageConfig | ObjectStorageConfig configures object storage in Thanos. | *[v1.SecretKeySelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#secretkeyselector-v1-core) | false |
+| ObjectStorageConfigFile | ObjectStorageConfigFile specifies the path of the object storage configuration file. Can't be used with ObjectStorageConfig at the same time. | *string | false |
```

Comment:
As mentioned on slack this should have a lower case start for this field, so camelCase not PascalCase.

Reply:
Thanks!

Fixed and pushed!